src/ServerPrivate.cc

```diff
@@ -584,6 +584,13 @@ sdf::Errors ServerPrivate::LoadSdfRootHelper(const ServerConfig &_config)
 {
   sdf::Errors errors;
 
+  sdf::ParserConfig sdfParserConfig = sdf::ParserConfig::GlobalConfig();
+  sdfParserConfig.SetStoreResolvedURIs(true);
+  sdfParserConfig.SetCalculateInertialConfiguration(
+    sdf::ConfigureResolveAutoInertials::SKIP_CALCULATION_IN_LOAD);
```

Comment:
should this be `SAVE_CALCULATION` instead of `SKIP_CALCULATION_IN_LOAD`?  @iche033

Reply:
actually, maybe not for meshes

Reply:
hmm yeah looks like mesh inertia should not be resolved in load.